pkg/utils/queue/delaying_queue_test.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package queue
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	fakeclock "k8s.io/utils/clock/testing"
+
+	"sigs.k8s.io/kwok/pkg/utils/wait"
+)
+
+func TestSimpleAddAfter(t *testing.T) {
+	fakeClock := fakeclock.NewFakeClock(time.Now())
+	dq := NewDelayingQueue[string](fakeClock)
+
+	first := "foo"
+	dq.AddAfter(first, 50*time.Millisecond)
+
+	// Simulate clock time passed
+	fakeClock.Step(10 * time.Millisecond)
+	if dq.Len() != 0 {
```

Comment:
I think the length here may never increase because there is no real sleep time.

If possible, I would like to add a check to see if the length increases given sleep time.

Reply:
Hi Shiming, I just added a  dedicated `checkIncreased` function for checking whether a queue is increased but with a sleep time given.